### ISSUE TITLE
Error on installing moodle 4.1 (php 8.1)

### ISSUE THREAD

**Issue by kirildevlion** [Vidalia/moodle-format_cards#7]
Hi we are trying to install plugin on MOODLE 4.1 (PHP 8.1) on 3 different instances and we got the same error.
When we deleted the file db/install.xml the plugin was successfully installed.
```
-->format_cards
Default exception handler: XML database file errors found Debug: Errors found in XMLDB file: PATH attribute does not match file directory: course/format/cards/db
Error code: ddlxmlfileerror
* line 1953 of /lib/upgradelib.php: call to upgrade_plugins()
* line 202 of /admin/cli/upgrade.php: call to upgrade_noncore()
 !!
```

**Comment by Vidalia** [comment]
Thanks for the report, I copied the install.xml from another plugin so I didn't have to type it out again but forgot to change one of the attributes, hence the error.

You need to install the plugin using install.xml because that sets up the database tables for you to be able to use section breaks. I'll push out a proper update shortly, but there's a diff for the patch here: https://github.com/Vidalia/moodle-format_cards/commit/751ba42981531b217d37d19208a2139585d27d92

**Comment by Vidalia** [comment]
Okay, should be fixed in v1.3.2 which is the new latest version on [moodle.org](https://moodle.org/plugins/format_cards/versions). Give that a try and let me know how it goes!
